Asset tracker v2 follow-up (#3879)
nrfconnect/sdk-nrf: applications/asset_tracker_v2/README.rst

The following configuration files are available in the application folder:

> * :file:`prj.conf` - Configuration file common for all build targets
> * :file:`boards/thingy91_nrf9160ns.conf` - Configuration file specific for Thingy:91, which is automatically merged with prj.conf when building for ``thingy91_nrf9160ns``
> * :file:`overlay-low-power.conf` - Configuration file to achieve the lowest power consumption, which disables features like LED control and logging that consume extra power

Reviewer: ```suggestion
* :file:`prj.conf` - Configuration file common for all build targets.
* :file:`boards/thingy91_nrf9160ns.conf` - Configuration file specific for Thingy:91. The file is automatically merged with :file:`prj.conf` when you build for the ``thingy91_nrf9160ns`` build target.
* :file:`overlay-low-power.conf` - Configuration file that achieve the lowest power consumption by disabling features  that consume extra power like LED control and logging.
```

Author: Fixed

* :file:`overlay-debug.conf` - Configuration file that adds additional verbose logging capabilities to the application

Generally, Kconfig overlays have an ``overlay-`` prefix and a ``.conf`` extension.

> Board-specific configuration files are placed in the ``boards`` folder, and are named ``<BOARD>.conf``.

Reviewer: ```suggestion
Board-specific configuration files are placed in the :file:`boards` folder and are named as :file:`<BOARD>.conf`.
```

Author: Fixed
